short/deploy-to-azure-from-github/README.md

```diff
@@ -0,0 +1,199 @@
+# Microsoft Student Summit
+## Launch into the Cloud with GitHub and Azure
+<hr>
+
+_"There is nothing that speaks more dev than having a recruiter ask for your resume and you point them to a url"_
+
+### Do you want to deploy your resume website for free on Azure? Here is  template to help you do that in a few steps.
+
+## Pre-requisites
+1. A GitHub Account
+1. An Azure Subscription. 🔊Students, you can sign up for the [free Azure for Students benefit here](https://azure.microsoft.com/en-us/free/students/?WT.mc_id=academic-0000-juliamuiruri)
+1. Attended Yuval's session to help you create a respoitory for this project
+
+## Steps
+1. With your respository open, click the . button on your keyboard.
+    This will open github.dev to allow you to edit the project on your browser.
+
+1. Open the `index.html` file in your repository, and paste in the code below. This is the html code responsible for the content on your resume website.
+
```

Comment:
These are fairly large, can you make them collapsible ? The way to do this is:

```html
<details>
   <summary><b>Some TITLE here</b></summary>
   Content here 
</details>
```

Reply:
Done!